packages/lesswrong/lib/collections/reviewWinnerArts/fragments.ts

```diff
@@ -5,7 +5,18 @@ export const ReviewWinnerArtImages = `
     splashArtImagePrompt
     splashArtImageUrl
     activeSplashArtCoordinates {
-      ...SplashArtCoordinates
+      ...SplashArtCoordinatesEdit
+    }
+  }
+`
+
+export const ReviewWinnerArtImagesForYear = `
```

Comment:
This fragment can be deleted (per https://github.com/ForumMagnum/ForumMagnum/pull/10585/files#r2025586562)